packages/inspection-report/src/components/CarView360/svgElementMapper/index.js

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable react/no-array-index-key */
 import PropTypes from 'prop-types';
 import React, { useMemo } from 'react';
-import { CommonPropTypes } from '../../resources';
+import { CommonPropTypes } from '../../../resources';
```

Comment:
is it valid? should it be only native changes?

Reply:
Yeah due to moving this file in folder this is a necessary update. And moving it to folder is required by linter rules cause I added native realisation for this file